nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -645,6 +645,11 @@ def test_plot_img_on_surf_hemispheres_and_orientations():
     plot_img_on_surf(nii,
                      hemispheres=['left', 'right'],
                      views=['dorsal', 'medial'])
+    # Check that manually set view angles work.
```

Comment:
this checks it doesn't raise an exception but could we check the view is set correctly?